packages/react/src/components/DataTable/TableBatchAction.js

```diff
@@ -14,9 +14,22 @@ const TableBatchAction = props => <Button {...props} />;
 
 TableBatchAction.propTypes = {
   /**
-   * Provide a text description for the icon in the button
+   * Specify if the button is an icon-only button
    */
-  iconDescription: PropTypes.string.isRequired,
+  hasIconOnly: PropTypes.bool,
```

Comment:
Is `hasIconOnly` being used anywhere?

Reply:
Oh right, it's not 😅 

I could update the batch actions story to include sample usage of `hasIconOnly` with `iconDescription`? Some like this, maybe for that last button in the batch actions series ("Download"):

```jsx
<TableBatchAction
  hasIconOnly
  renderIcon={Download}
  iconDescription="Download the selected rows"
  onClick={batchActionClick(selectedRows)}
/>
```

Or would you prefer to remove `hasIconOnly` from this?